client/src/main/java/org/apache/celeborn/client/ShuffleClient.java

```diff
@@ -85,17 +84,10 @@ public static FileSystem getHdfsFs(CelebornConf conf) {
     if (null == hdfsFs) {
       synchronized (ShuffleClient.class) {
         if (null == hdfsFs) {
-          Configuration hdfsConfiguration = CelebornHadoopUtils$.MODULE$.newConfiguration(conf);
-          // enable fs cache to avoid too many fs instances
-          hdfsConfiguration.set("fs.hdfs.impl.disable.cache", "false");
-          hdfsConfiguration.set("fs.viewfs.impl.disable.cache", "false");
-          logger.info(
-              "Celeborn client will ignore cluster"
-                  + " settings about fs.hdfs/viewfs.impl.disable.cache and set it to false");
           try {
-            hdfsFs = FileSystem.get(hdfsConfiguration);
-          } catch (IOException e) {
-            System.err.println("Celeborn initialize hdfs failed.");
+            hdfsFs = CelebornHadoopUtils.getHadoopFS(conf);
+          } catch (Exception e) {
+            System.err.println("Celeborn initialize HDFS failed.");
             e.printStackTrace(System.err);
```

Comment:
why not use `logError`?

Reply:
Except to style, some additional background: we internally use log4j to send logs messages to the remote log service, but the stdout/stderr are not included.

Reply:
This code will execute in the spark executor. If it is running on yarn, the system.err will output to stderr.

Reply:
In any case, what's wrong to use logError?

Reply:
It can use logError. I'll change it.